spinalcordtoolbox/deepseg/models.py

```diff
@@ -400,13 +400,14 @@
              ```"""),  # noqa E501 (line too long)
          'group': 'pathology'
          },
-    'rootlets_t2':
-        {'description': 'Segmentation of spinal nerve rootlets for T2w contrast',
+    'rootlets':
+        {'description': 'Segmentation of spinal nerve rootlets for T2w and MP2RAGE (UNIT1, INV1, INV2) images',
          'long_description': 'This segmentation model for spinal nerve rootlets segmentation uses a 3D U-Net '
                              'architecture, and was trained with the nnUNetV2 framework. It is a multiclass model, '
-                             'outputting a single segmentation image containing 8 classes representing the C2-C8 '
-                             'dorsal spinal cord nerve rootlets. Training data consisted of 31 isotropic T2w images '
-                             'from healthy subjects from two different open-access datasets.',
+                             'outputting a single segmentation image containing 8 classes representing the C2-T1 '
+                             'dorsal and ventral spinal cord nerve rootlets. Training data consisted of 76 isotropic '
+                             'T2w and MP2RAGE images from healthy subjects from three datasets (spine-generic '
+                             'multi-subject, OpenNeuro ds004507, 7T MP2RAGE (private)).',
```

Comment:
'outputting a single segmentation image containing 8 classes representing the C2-T1 '
                             'dorsal and ventral spinal cord nerve rootlets. Training data consisted of images from healthy participants from three datasets: spine-generic '
                             'multi-subject (3T T2w: n=XX), OpenNeuro ds004507 (3T T2w, n=XX), private data (7T MP2RAGE (n= XX)).',